darwin/cli.py

```diff
@@ -167,6 +167,10 @@ def _run(args: Namespace, parser: ArgumentParser) -> None:
                 args.w,
                 args.h,
             )
+    elif args.command == "benchmarks":
```

Comment:
Is there to be any other benchmarks command - if not `benchmarks-run` might be more appropriate, but if it's just future proofing, np.

Reply:
I don't think there are any immediate plans, but I can imagine something like `benchmarks status <id>` or `benchmarks get-results <id>`.